lona/html/attribute_dict.py

```diff
@@ -1,5 +1,7 @@
 from lona.protocol import PATCH_TYPE, OPERATION
 
+BOOLEAN_ATTRIBUTES = ['checked', 'selected', 'disabled']
```

Comment:
I'm not sure list is complete.

Reply:
Good point. I didn't found a official list of all boolean attributes, but this list seems to be accurate
https://meiert.com/en/blog/boolean-attributes-of-html/

Reply:
Seems it almost matches with this list https://html.spec.whatwg.org/#attributes-3

Reply:
@maratori I will use the table from your link

Reply:
@maratori done